zebra-consensus/src/primitives/ed25519.rs

```diff
@@ -11,7 +11,6 @@ use futures::{future::BoxFuture, FutureExt};
 use once_cell::sync::Lazy;
```

Comment:
I'm making this comment to block merging this PR until we're sure it's needed for the stable release, and there's a test plan.

(We can test after merging and revert if there are any issues.)

Reply:
Since we've made a few stable releases now, I think it's ok to merge this PR without a test plan.